composer.json

```diff
@@ -34,7 +34,7 @@
         "ircmaxell/password-compat": "1.0.4",
         "ircmaxell/random-lib": "*",
         "shudrum/array-finder": "1.1.0",
-        "phpoffice/phpexcel": "1.8",
+        "phpoffice/phpexcel": "1.8.1",
```

Comment:
Good idea, however we can do even better by not setting a specific version anymore.

We could have the latest version available, but without backward compatibility issues, by writing this line:
```
"phpoffice/phpexcel": "~1.8",
```